src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs

```diff
@@ -0,0 +1,1919 @@
+        /// <exception cref="InvalidOperationException">This exception can be due to invalid state.</exception>
+        /// <pre>
+        /// The State should be Ready
+        /// </pre>
+        public static void ClearCommandList()
+        {
+            ErrorCode error = VcMgrUnsetCommandList();
+            if (error != ErrorCode.None)
+            {
+                Log.Error(LogTag, "ClearCommandList Failed with error " + error);
+                throw ExceptionFactory.CreateException(error);
+            }
+        }
+
+        /// <summary>
+        /// Sets commands from file.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>http://tizen.org/privilege/voicecontrol.manager</privilege>
+        /// <privlevel>public</privlevel>
+        /// <feature>http://tizen.org/feature/speech.control</feature>
+        /// <feature>http://tizen.org/feature/microphone</feature>
+        /// <param name="path">File Path</param>
+        /// <param name="type">Command type</param>
+        /// <exception cref="InvalidOperationException">
+        /// This can occur due to the following reasons:
+        /// 1) This exception can be due to an invalid state.
+        /// 2) This exception can be due to operation failed.
+        /// </exception>
+        /// <exception cref="ArgumentException">This exception can be due to an invalid parameter.</exception>
+        /// <pre>
+        /// The State should be Ready
+        /// </pre>
+        public static void SetCommandListFromFile(string path, CommandType type)
+        {
+            ErrorCode error = VcMgrSetCommandListFromFile(path, type);
+            if (error != ErrorCode.None)
+            {
+                Log.Error(LogTag, "SetCommandListFromFile Failed with error " + error);
+                throw ExceptionFactory.CreateException(error);
+            }
+
+        }
+
+        /// <summary>
+        /// Retrieves all available commands.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>http://tizen.org/privilege/voicecontrol.manager</privilege>
+        /// <privlevel>public</privlevel>
+        /// <feature>http://tizen.org/feature/speech.control</feature>
+        /// <feature>http://tizen.org/feature/microphone</feature>>
+        /// <returns>
+        /// The Command List else null in case of no System Commands
+        /// </returns>
+        /// <exception cref="InvalidOperationException">
+        /// This can occur due to the following reasons:
+        /// 1) This exception can be due to an invalid state.
+        /// 2) This exception can be due to operation failed.
+        /// </exception>
+        /// <exception cref="ArgumentException">This exception can be due to an invalid parameter.</exception>
+        /// <pre>
+        /// The State should be Ready
+        /// </pre>
+        public static VoiceCommandsGroup GetCurrentCommands()
```

Comment:
Need to cache the `VoiceCommandsGroup` who is set by `SetCommandList(VoiceCommandsGroup list)` and return it. 

if not, it has issue below test code
``` c#
var group1 = new VoiceCommandsGroup();
...
VoiceControlManager.SetCommandList(group1);
var group2 = VoiceControlManager.GetCurrentCommands();

//group1.Commands contains same commands with group2.Commands
//but, group1 == group2 it is false

//after add command in group1
group1.Commands.Add(....);
//group1.Commands contains different commands with group2.Commands even though it has same native handle



```

Reply:
I will change VoiceCommandsGroup type to IEnumerable<VoiceCommand>.

Reply:
Fixed. please check a new patch.